tools/misc/browse_dataset.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+from mmcv import Config, mkdir_or_exist, track_iter_progress
+from os import path as osp
+
+from mmdet3d.core.bbox import Box3DMode, Coord3DMode
+from mmdet3d.core.visualizer.open3d_vis import Visualizer
+from mmdet3d.datasets import build_dataset
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Browse a dataset')
+    parser.add_argument('config', help='train config file path')
```

Comment:
The arguments and functionality in mmdet are necessary and should be kept https://github.com/open-mmlab/mmdetection/blob/master/tools/misc/browse_dataset.py#L16